pulp_rpm/app/tasks/copy.py

```diff
@@ -91,15 +91,15 @@ def find_children_of_content(content, src_repo_version):
             packagegroups = packagegroups.union(category_package_groups)
 
     for packageenvironment in packageenvironments.iterator():
-        for env_package_group in packageenvironment.packagegroups:
+        for env_package_group in packageenvironment.group_ids:
             env_package_groups = PackageGroup.objects.filter(
                 name=env_package_group['name'], pk__in=src_repo_version.content
             )
             children.update(
                 [envgroup.pk for envgroup in env_package_groups]
             )
             packagegroups = packagegroups.union(env_package_groups)
-        for optional_env_package_group in packageenvironment.optionalgroups:
+        for optional_env_package_group in packageenvironment.option_ids:
             opt_env_package_groups = PackageGroup.objects.filter(
                 name=optional_env_package_group['name'], pk__in=src_repo_version.content
```

Comment:
Does ```.option_ids``` and ```.group_ids``` return dictionaries?

Reply:
it does. here is an example of simple `pkgenv.group_ids`

`[{'name': 'lion', 'default': False}]`